### PR TITLE
Upgrade PHP version

### PR DESCRIPTION
**Pull request by madhur**
Fixes https://github.com/CachetHQ/Docker/issues/213

### CODE REVIEW

**Comment by wfjsw** [comment]
Hmm...why docker engine

**Comment by wfjsw** [comment]
Time to upgrade docker version lock too?

CC @djdefi

File: Dockerfile [hunk 1]
```diff
@@ -9,6 +9,11 @@ ENV cachet_ver ${cachet_ver:-master}
 
 ENV COMPOSER_VERSION 1.4.1
 
+ADD https://php.codecasts.rocks/php-alpine.rsa.pub /etc/apk/keys/php-alpine.rsa.pub
```
Comment (djdefi) [inline Dockerfile]: Is there any information on what this `php.codecasts.rocks` source is? I am unfamiliar with it, and am not really able to find much info on it. Can't we use an official Alpine or PHP repository?

**Comment by djdefi** [comment]
I also pushed a fix to master that should get CI happy again. Will need to merge in the upstream changes to this branch for that to go green.

File: Dockerfile (hunk 1 above)
Comment (madhur) [inline Dockerfile]: I have got this reference from https://github.com/codecasts/php-alpine

**Comment by djdefi** [comment]
I pushed an update to the base image in https://github.com/CachetHQ/Docker/pull/217 which I think may address this issue via newer working upstream php packages.

**Comment by djdefi** [comment]
Feel free to reopen and discuss if the changes in #217 did not help this.

**Pull request closed without merging**